src/knx/address_table_object.cpp

```diff
@@ -38,11 +38,11 @@ uint16_t AddressTableObject::getTsap(uint16_t addr)
 {
     uint16_t size = entryCount();
     #ifdef USE_BINSEARCH
-    addr = htons(addr);
+    addr = ntohs(addr);
```

Comment:
here i would prefer to keep htons because it is the logically correct conversion.
addr is in host-byte-order, the table in network-byte-order.
so addr has to be converted from host to network to compare.

Reply:
done.